setup.py

```diff
@@ -1,11 +1,24 @@
+#!/usr/bin/env python
+# Copyright (c) 2017 "Shopify inc." All rights reserved.
+# Use of this source code is governed by a MIT-style license that can be found in the LICENSE file.
+import re
+
 try:
-    from setuptools import setup
+    import setuptools as setuplib
```

Comment:
Any reason for this to be conditional? Not the least because you are apparently depending on it now.